### PR TITLE
translated /about/community.md page to Ukrainian

### PR DESCRIPTION
**Pull request by alexandrtovmach**
Help with the localization process:

- translated `/about/community.md` page to Ukrainian

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-uk: Please have a review, thanks!

**Comment by alexandrtovmach** [comment]
Thank you, but unfortunately it seems like stalled team, so I've created direct PR, and now working on other translations to Ukrainian and Russian

File: locale/uk/about/community.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Якщо Ви зацікавлені в участі в CommComm, як член комітету, вам слід прочитати розділ нижче ― про **Спостерігачів та Членство**, і створити issue з проханням бути спостерігачем, на нашому наступному засіданні CommComm. Ознайомитись з прикладом такого питання можна [тут](https://github.com/nodejs/community-committee/issues/142).
+
+## Спостерігачі та Членство
+
+Якщо ви зацікавлені в більш глибокій взаємодії з CommComm та його проектами, ми заохочуємо вас стати активним Спостерігачем і працювати над досягненням статусу Члена. Для цього необхідно:
+
+ 1. Відвідувати зустрічі, що відбуваються двічі на тиждень, досліджувати issues, позначені як "Good first issue", обробляти issues та pull requests, а також надати інформацію про себе, як учасник чи співавтор, через GitHub.
+ 2. Створити запит на право стати Спостерігачем шляхом створення issue. Після того, як до нього приєднаються інші Спостерігачі, ми будемо відслідковувати вашу відвідуваність і участь протягом 3 місяців, відповідно до наших принципів. Ви можете знайти чудовий приклад такого питання [тут](https://github.com/nodejs/community-committee/issues/142).
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]: If I am not mistaken:
```suggestion
 2. Створити запит на право стати Спостерігачем шляхом створення issue. Після того, як ви приєднаєтесь в якості Спостерігача, ми будемо відслідковувати вашу відвідуваність і участь протягом 3 місяців, відповідно до наших принципів. Ви можете знайти чудовий приклад такого питання [тут](https://github.com/nodejs/community-committee/issues/142).
```

File: locale/uk/about/community.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+Далі, ви можете брати участь у наших починаннях, пов'язаних з побудовою спільноти, таких як: локалізація, євангелізація, Node.js Collection та інші ― копаючись у репозиторіях і беручи участь!
+
+Та перед тим як почати, будь-ласка озайомтесь з [рекомендаціями](https://github.com/nodejs/community-committee/blob/master/governance/COLLABORATOR_GUIDE.md).
+
+Якщо Ви зацікавлені в участі в CommComm, як член комітету, вам слід прочитати розділ нижче ― про **Спостерігачів та Членство**, і створити issue з проханням бути спостерігачем, на нашому наступному засіданні CommComm. Ознайомитись з прикладом такого питання можна [тут](https://github.com/nodejs/community-committee/issues/142).
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]: Casing consistency and comma nits:
```suggestion
Якщо ви зацікавлені в участі в CommComm як член комітету, вам слід прочитати розділ нижче ― про **Спостерігачів та Членство**, і створити issue з проханням бути спостерігачем на нашому наступному засіданні CommComm. Ознайомитись з прикладом такого питання можна [тут](https://github.com/nodejs/community-committee/issues/142).
```

File: locale/uk/about/community.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Спостерігачі та Членство
+
+Якщо ви зацікавлені в більш глибокій взаємодії з CommComm та його проектами, ми заохочуємо вас стати активним Спостерігачем і працювати над досягненням статусу Члена. Для цього необхідно:
+
+ 1. Відвідувати зустрічі, що відбуваються двічі на тиждень, досліджувати issues, позначені як "Good first issue", обробляти issues та pull requests, а також надати інформацію про себе, як учасник чи співавтор, через GitHub.
+ 2. Створити запит на право стати Спостерігачем шляхом створення issue. Після того, як до нього приєднаються інші Спостерігачі, ми будемо відслідковувати вашу відвідуваність і участь протягом 3 місяців, відповідно до наших принципів. Ви можете знайти чудовий приклад такого питання [тут](https://github.com/nodejs/community-committee/issues/142).
+ 3. Якщо ви задовольняєте мінімальну відвідуваність за 3 місяці та все ще хочете приймати участь, CommComm проголосує, щоб додати Вас в якості Члена.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]: For consistency:
```suggestion
 3. Якщо ви задовольняєте мінімальну відвідуваність за 3 місяці та все ще хочете приймати участь, CommComm проголосує, щоб додати вас в якості Члена.
```

File: locale/uk/about/community.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+Та перед тим як почати, будь-ласка озайомтесь з [рекомендаціями](https://github.com/nodejs/community-committee/blob/master/governance/COLLABORATOR_GUIDE.md).
+
+Якщо Ви зацікавлені в участі в CommComm, як член комітету, вам слід прочитати розділ нижче ― про **Спостерігачів та Членство**, і створити issue з проханням бути спостерігачем, на нашому наступному засіданні CommComm. Ознайомитись з прикладом такого питання можна [тут](https://github.com/nodejs/community-committee/issues/142).
+
+## Спостерігачі та Членство
+
+Якщо ви зацікавлені в більш глибокій взаємодії з CommComm та його проектами, ми заохочуємо вас стати активним Спостерігачем і працювати над досягненням статусу Члена. Для цього необхідно:
+
+ 1. Відвідувати зустрічі, що відбуваються двічі на тиждень, досліджувати issues, позначені як "Good first issue", обробляти issues та pull requests, а також надати інформацію про себе, як учасник чи співавтор, через GitHub.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]: As per "bi-monthly" below and Node.js Calendar, it seems to be rather:
```suggestion
 1. Відвідувати зустрічі, що відбуваються раз на два тижні, досліджувати issues, позначені як "Good first issue", обробляти issues та pull requests, а також надати інформацію про себе, як учасник чи співавтор, через GitHub.
```

File: locale/uk/about/community.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+ 2. Створити запит на право стати Спостерігачем шляхом створення issue. Після того, як до нього приєднаються інші Спостерігачі, ми будемо відслідковувати вашу відвідуваність і участь протягом 3 місяців, відповідно до наших принципів. Ви можете знайти чудовий приклад такого питання [тут](https://github.com/nodejs/community-committee/issues/142).
+ 3. Якщо ви задовольняєте мінімальну відвідуваність за 3 місяці та все ще хочете приймати участь, CommComm проголосує, щоб додати Вас в якості Члена.
+
+Членство надається на 6 місяців. Група буде регулярно цікавитись, про бажання залишити членство. Учаснику потрібно лише надати згоду на поновлення. Немає фіксованого розміру CommComm. Проте, очікувана ціль становить від 9 до 12 учасників. Ви можете прочитати більше про членство та інші адміністративні деталі в нашому [керівництві з управління](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md).
+
+Регулярні зустрічі CommComm проводяться щомісяця, на відеоконференції Zoom і публікуються в прямому ефірі на YouTube. Будь-який член спільноти, або учасник, може попросити щось додати до порядку денного наступної зустрічі, за допомогою GitHub Issues.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
Регулярні зустрічі CommComm проводяться двічі на місяць на відеоконференції Zoom і публікуються в прямому ефірі на YouTube. Будь-який член спільноти або учасник може попросити щось додати до порядку денного наступної зустрічі за допомогою GitHub Issues.
```

File: locale/uk/about/community.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+Оголошення про зустрічі та порядок денний публікуються до початку зустрічі в [GitHub issues](https://github.com/nodejs/community-committee/issues). Ви також можете знайти розклад зустрічей у [Node.js календарі](https://nodejs.org/calendar). Для того, щоб дивитись онлайн-трансляції по Node.js на YouTube, підпишіться на [Node.js Foundation YouTube channel](https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ). Не забудьте натиснути дзвінок, щоб отримувати сповіщення про нові відео!
+
+## Процес пошуку консенсусу
+
+CommComm слідує наступній [моделі пошуку консенусу](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making), для їх вирішення.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]: Maybe:
```suggestion
В разі виникнення розбіжностей, CommComm слідує наступній [моделі пошуку консенсусу](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making).
```

File: locale/uk/about/community.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+
+Коли пункт порядку денного виявився досягнутим консенсусу, модератор запитає: "Чи хто-небудь заперечує?", як останній заклик до незгоди з консенсусом. Якщо не може бути досягнуто консенсусу, який не має заперечень, то називається більшість голосів. Очікується, що більшість рішень, прийнятих CommComm, здійснюється через процес пошуку консенсусу, і що голосування використовується лише як останнє рішення.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
Коли пункт порядку денного виявився досягнутим консенсусу, модератор запитає: "Чи хто-небудь заперечує?", як останній заклик до незгоди з консенсусом. Якщо не може бути досягнуто беззаперечного консенсусу, питання виноситься на голосування, в якому перемагає більшість голосів. Очікується, що більшість рішень, прийнятих CommComm, здійснюється через процес пошуку консенсусу і що голосування використовується лише як останній засіб.
```

File: locale/uk/about/community.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+## Учасники та Співробітники
+
+Завданням CommComm є подальша підтримка та розширення спільноти Node.js. Якщо ви читаєте це, ви вже є частиною спільноти ― і як частина спільноти Node.js, ми хотіли б отримати вашу допомогу!
+
+Краще всього почати з [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub репозиторію. Спробуйте знайти [завдання помічені, як "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue), щоб зрозуміти де ми потребуємо допомоги. Якщо у вас є власні ідеї про те, як ми можемо покращити спільноту, не соромтеся відкривати свої власні завдання, створювати pull requests з покращеннями, або допомагайте нам, ділячись своїми думками та ідеями в поточних дискусіях, які вже є на GitHub.
+
+Далі, ви можете брати участь у наших починаннях, пов'язаних з побудовою спільноти, таких як: локалізація, євангелізація, Node.js Collection та інші ― копаючись у репозиторіях і беручи участь!
+
+Та перед тим як почати, будь-ласка озайомтесь з [рекомендаціями](https://github.com/nodejs/community-committee/blob/master/governance/COLLABORATOR_GUIDE.md).
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
Та перед тим як почати, будь ласка, ознайомтесь з [рекомендаціями](https://github.com/nodejs/community-committee/blob/master/governance/COLLABORATOR_GUIDE.md).
```

File: locale/uk/about/community.md [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+ - Переклад та інтернаціоналізація
+ - Модерація та підтримка стабільності проекту
+ - Пропаганда та [Публікація](https://medium.com/the-node-js-collection)
+
+Є чотири типи участі в CommComm:
+
+  - **Учасник (Contributor)** ― це будь-яка особа, яка створює/коментує issue або pull request.
+  - **Співробітник (Collaborator)** ― це учасник, якому було надано права до запису до репозиторію.
+  - **Спостерігач (Observer)** ― це будь-яка особа, яка запросила або отримала запит від CommComm. Це також перший крок до членства.
+  - **Член (Member)** ― це співробітник з правом голосу, який виконав вимоги, щодо участі, і був запрошений до процесу голосування CommComm.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
  - **Член (Member)** ― це співробітник з правом голосу, який виконав вимоги щодо участі і був запрошений до процесу голосування CommComm.
```

File: locale/uk/about/community.md [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+
+Більше деталей про членство у CommComm можна прочитати [тут](https://github.com/nodejs/community-committee).
+
+## Учасники та Співробітники
+
+Завданням CommComm є подальша підтримка та розширення спільноти Node.js. Якщо ви читаєте це, ви вже є частиною спільноти ― і як частина спільноти Node.js, ми хотіли б отримати вашу допомогу!
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
Завданням CommComm є подальша підтримка та розширення спільноти Node.js. Якщо ви читаєте це, ви вже є частиною спільноти ― і, як частина спільноти Node.js, ми хотіли б отримати вашу допомогу!
```

**Comment by ghost** [comment]
@alexandrtovmach：For Russian version, if this is different from Ukrainian, you have to create a new locale by copying 'en' folder with all of its files, and then do translations for them（Maybe the language name will be 'ru'？）

The reason why I help you to notify your group is because some member (such as @vsemozhetbyt or someone else) will help you to have a review if he/she is free :) 

So no worries about that, but still say a BIG THANK YOU to you for your first contribution!

File: locale/uk/about/community.md [hunk 11]
```diff
@@ -0,0 +1,56 @@
...
+
+Краще всього почати з [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub репозиторію. Спробуйте знайти [завдання помічені, як "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue), щоб зрозуміти де ми потребуємо допомоги. Якщо у вас є власні ідеї про те, як ми можемо покращити спільноту, не соромтеся відкривати свої власні завдання, створювати pull requests з покращеннями, або допомагайте нам, ділячись своїми думками та ідеями в поточних дискусіях, які вже є на GitHub.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]: Some comma nits:
```suggestion
Краще всього почати з [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub репозиторію. Спробуйте знайти [завдання, помічені як "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue), щоб зрозуміти, де ми потребуємо допомоги. Якщо у вас є власні ідеї про те, як ми можемо покращити спільноту, не соромтеся відкривати свої власні завдання, створювати pull requests з покращеннями, або допомагайте нам, ділячись своїми думками та ідеями в поточних дискусіях, які вже є на GitHub.
```

File: locale/uk/about/community.md [hunk 12]
```diff
@@ -0,0 +1,56 @@
...
+
+ 1. Відвідувати зустрічі, що відбуваються двічі на тиждень, досліджувати issues, позначені як "Good first issue", обробляти issues та pull requests, а також надати інформацію про себе, як учасник чи співавтор, через GitHub.
+ 2. Створити запит на право стати Спостерігачем шляхом створення issue. Після того, як до нього приєднаються інші Спостерігачі, ми будемо відслідковувати вашу відвідуваність і участь протягом 3 місяців, відповідно до наших принципів. Ви можете знайти чудовий приклад такого питання [тут](https://github.com/nodejs/community-committee/issues/142).
+ 3. Якщо ви задовольняєте мінімальну відвідуваність за 3 місяці та все ще хочете приймати участь, CommComm проголосує, щоб додати Вас в якості Члена.
+
+Членство надається на 6 місяців. Група буде регулярно цікавитись, про бажання залишити членство. Учаснику потрібно лише надати згоду на поновлення. Немає фіксованого розміру CommComm. Проте, очікувана ціль становить від 9 до 12 учасників. Ви можете прочитати більше про членство та інші адміністративні деталі в нашому [керівництві з управління](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md).
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
Членство надається на 6 місяців. Група буде регулярно цікавитись про бажання залишити членство. Учаснику потрібно лише надати згоду на поновлення. Немає фіксованого розміру CommComm. Проте, очікувана ціль становить від 9 до 12 учасників. Ви можете прочитати більше про членство та інші адміністративні деталі в нашому [керівництві з управління](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md).
```

**Review by vsemozhetbyt — APPROVED** [review]
Thank you!
Just some possible nits.

**Comment by Maledong** [comment]
@alexandrtovmach：Now you can follow suggestions from the reviewer and do changes for your submit :)

**Comment by alexandrtovmach** [comment]
I'll apply it in a few days. Thanks for a help.

And thanks for notes about Russian translation, I'm doing in the same way. Thanks a lot

**Comment by ghost** [comment]
@alexandrtovmach：Just a tip——The 'knowledge' and all the files in 'blog' you don't need to do any translations now.

**Comment by alexandrtovmach** [comment]
Sure, thanks

**Comment by alexandrtovmach** [comment]
applied @vsemozhetbyt notes

File: locale/uk/about/community.md [hunk 13]
```diff
@@ -0,0 +1,56 @@
...
+
+Регулярні зустрічі CommComm проводяться двічі на місяць на відеоконференції Zoom і публікуються в прямому ефірі на YouTube. Будь-який член спільноти, або учасник, може попросити щось додати до порядку денного наступної зустрічі, за допомогою GitHub Issues.
+
+Оголошення про зустрічі та порядок денний публікуються до початку зустрічі в [GitHub issues](https://github.com/nodejs/community-committee/issues). Ви також можете знайти розклад зустрічей у [Node.js календарі](https://nodejs.org/calendar). Для того, щоб дивитись онлайн-трансляції по Node.js на YouTube, підпишіться на [Node.js Foundation YouTube channel](https://www.youtube.com/channel/UCQPYJluYC_sn_Qz_XE-YbTQ). Не забудьте натиснути дзвінок, щоб отримувати сповіщення про нові відео!
+
+## Процес пошуку консенсусу
+
+В разі виникнення розбіжностей, CommComm слідує наступній [моделі пошуку консенусу](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making), для їх вирішення.
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
В разі виникнення розбіжностей, CommComm слідує наступній [моделі пошуку консенсусу](https://en.wikipedia.org/wiki/Consensus-seeking_decision-making), для їх вирішення.
```

File: locale/uk/about/community.md [hunk 14]
```diff
@@ -0,0 +1,56 @@
...
+
+## Учасники та Співробітники
+
+Завданням CommComm є подальша підтримка та розширення спільноти Node.js. Якщо ви читаєте це, ви вже є частиною спільноти ― і, як частина спільноти Node.js, ми хотіли б отримати вашу допомогу!
+
+Краще всього почати з [nodejs/community-committee](https://github.com/nodejs/community-committee) GitHub репозиторію. Спробуйте знайти [завдання, помічені як "Good first issue"](https://github.com/nodejs/community-committee/labels/good%20first%20issue), щоб зрозуміти, де ми потребуємо допомоги. Якщо у вас є власні ідеї про те, як ми можемо покращити спільноту, не соромтеся відкривати свої власні завдання, створювати pull requests з покращеннями, або допомагайте нам, ділячись своїми думками та ідеями в поточних дискусіях, які вже є на GitHub.
+
+Далі, ви можете брати участь у наших починаннях, пов'язаних з побудовою спільноти, таких як: локалізація, євангелізація, Node.js Collection та інші ― копаючись у репозиторіях і беручи участь!
+
+Та перед тим як почати, будь ласка, озайомтесь з [рекомендаціями](https://github.com/nodejs/community-committee/blob/master/governance/COLLABORATOR_GUIDE.md).
```
Comment (vsemozhetbyt) [inline locale/uk/about/community.md]:
```suggestion
Та перед тим як почати, будь ласка, ознайомтесь з [рекомендаціями](https://github.com/nodejs/community-committee/blob/master/governance/COLLABORATOR_GUIDE.md).
```

**Review by vsemozhetbyt — APPROVED** [review]

**Comment by ghost** [comment]
@vsemozhetbyt：Can I merge this now?

**Comment by vsemozhetbyt** [comment]
Yes, I think)

**Comment by ghost** [comment]
Thanks @alexandrtovmach  and @vsemozhetbyt for your contributions!

**Pull request merged**